services/httpd/handler.go

```diff
@@ -929,7 +930,7 @@ func (h *Handler) recovery(inner http.Handler, name string) http.Handler {
 		defer func() {
 			if err := recover(); err != nil {
 				logLine := buildLogLine(l, r, start)
-				logLine = fmt.Sprintf(`%s [panic:%s]`, logLine, err)
```

Comment:
I'm not sure we even need the wrapping `%s [panic ]`. I would vote to just dump `debug.Stack()` as-is.